mixins/localize-mixin.md

```diff
@@ -121,6 +121,49 @@ If your localized string contains arguments, pass them as a key-value object as
 
 ```javascript
 render() {
-  return html`<p>${this.localize('hello', {firstName: 'Mary'})}</p>`;
+  return html`<p>${this.localize('hello', { firstName: 'Mary' })}</p>`;
 }
 ```
+
+## `localizeHTML()`
+
+You can include some basic markup in your localization resources and safely convert them to HTML with the `localizeHTML()` method
+
+### Simple Markup
+
+#### `[b]Bold[/b]`
+The `[b]` markup converts to the `<strong>` tag
+
+#### `[i]Italic[/i]`
+The `[i]` markup converts to the `<em>` tag
+
+#### `[br]`
+The `[br]` markup converts to the `<br />` tag
+
+
+### Complex markup
+
+Complex markup is more powerful, but requires passing additional information in the second parameter
+
+#### `[a]Linked[/a]`
+The `[a]` markup converts to the `<d2l-link>` tag, and takes an attributes string in the `_links` property
+
+#### `[tt]Tooltipped[/tt]`
+The `[tt]` markup converts to the `<d2l-tooltip-help>` tag, and takes tooltip text in the `_tooltips` property
```

Comment:
`<tt>` is [already an element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tt), so I think we'll want to go with something like `[tooltip-help]` here to be explicit.

Reply:
Oof, yeah. I'm leaning toward the more contextually-helpful names now.